CIP-0013/CIP-0013.md

```diff
@@ -43,27 +55,66 @@ Example:
+poolhexid = 56HEXDIG
+poolticker = 3*5UNICODE
 ```
 
+### Payment URI queries
+
 The amount parameter must follow the [same rules](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#transfer-amountsize) described in BIP-21, namely, it must be specified in decimal ADA, without commas and using the period (.) decimal separator.
 
+### Stake pool URI queries
+
+For brevity, essential in many Internet contexts, `poolticker`  must be supported here in addition to the unambiguous `poolhexid`.
+
+When there is more than one pool registered with the specified `poolTicker` (whether for pool groups which have the same ticker for all pools, or for separate pools using the same ticker), the choice to which pool to finally delegate is left to the user through the wallet UI.
+
+### Handling stake pool links
+
+The wallet UI should always confirm the exact delegation choice even when it is unambiguous from the URI.  When the user has multiple wallets, the wallet UI must select which wallet(s) the user will be delegating from.
+
+These yet unsupported parameters in the URI query string should (by preference of the wallet UI designers) *either* be ignored *or* generate a warning message, to avoid leading the user to believe they are implementing a currently unsupported but perhaps popularly referenced multi-pool delegation list:
+
+* any value for the first URI query argument;
+* any URI query argument beyond the first.
+
 ## Security Considerations
 
-1. We cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
-2. We should be wary of people who disguise “donate” links as actually opening up a phishing website that LOOKS like a wallet.
+1. For payment links, we cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
+2. For either payment or staking links, we should be wary of people who disguise links as actually opening up a phishing website that LOOKS like that corresponding part of the wallet UI.
 
 # Rationale
 
 ## Why not use Universal links, deep links and other non-protocol-based Solution
 
 An alternative solution to the original problem described above is to use standard URL links in combination with a routing backend system. The routing system is used to redirect to the app's URI. The advantage of this scheme is that it allows to provide a fallback mechanism to handle the case when no application implementing the protocol is installed (for instance, by redirecting to the App Store or Google Play). This is the approach behind iOS Universal Links and Android App Links. In general, it provides a better user experience but requires a centralized system which makes it unsuitable for as a multi-app standard.
 
+## Why stake pool links are important
```

Comment:
I think it's better if we keep the "rationale" section related to technical rationales and not the motivation for the proposal (since we already have a "motivation" section for this reason)

Reply:
These first two points are therefore moved into the Motivation section:

- Interfaces that connect delegators with pools beyond the highly contested top choices of the in-wallet ranking algorithms are important to avoid saturation and maintain decentralization.
- Larger pools and collectives can also use these URIs to link to a family of pools they own to avoid any one of their pools becoming saturated.

leaving this item which poses a technical advantage of stake pool links versus some other means of inter-process communication placing more of the burden on the wallet:

- Pool links allow for interfaces to initiate delegation transactions without requiring any code modifications to the wallets themselves.

Also the mostly eliminated section no longer needs its own title since it relates to the general comment about links as they apply also to payment URIs, so the title **Why stake pool links are important** has also been eliminated.